### ISSUE TITLE
gitpod extension (maybe a new one): allow integration with vscode.dev

### ISSUE THREAD

**Issue by GitMensch** [gitpod-io/openvscode-server#216]
There are currently two gitpod extensions in the ms extension market, I think I fail to see from the README what they are actually for (the README seems to be about Gitpod, not about the extension).
That aside: It would be cool if there is an extension that allows using the existing gitpot workspaces in vscode.dev (providing Terminal/build/debug).

**Comment by akosyakov** [comment]
I don't think it is technically feasible. There is API between web workbench (vscode.dev) and VS Code Server running on the backend. It is not stable, i.e. it is only safe when web workbench and server are built from the same commit. Connecting arbitrary one likely is not going work and will break in subtle ways.
